.gitignore

```diff
@@ -0,0 +1 @@
+.tmp/
```

Comment:
Remove?

Reply:
If possible, put in a CI-specific directory

Reply:
Done!